types/cheesy-api/package.json

```diff
@@ -0,0 +1,4 @@
+{
+	"name": "cheesy-api-types",
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/cheesy-api/package.json;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.